README.md

```diff
@@ -11,6 +11,7 @@ Add the latest version of `coveralls` to your package.json:
 ```
 npm install coveralls --save-dev
 ```
+The command (`coveralls`) can be used from the command line.
```

Comment:
Not true, because `coveralls` won't be installed globally by the command above:

```
npm install coveralls --save-dev
```

Reply:
I do think it makes sense to add a footnote or additional detail that sometimes `coveralls`  does exist in the path. For example, on travis CI build jobs they append `node_modules/.bin` to the `$PATH` so you can invoke `coveralls` from your `.travis.yml`. Relevant build log on nodejs travis log:
```sh
 export PATH=./node_modules/.bin:$PATH
```